core/field_dropdown.js

```diff
@@ -58,6 +58,7 @@ Blockly.FieldDropdown = function(menuGenerator, opt_validator) {
     Blockly.FieldDropdown.validateOptions_(menuGenerator);
   }
   this.menuGenerator_ = menuGenerator;
+  this.selectedMenuItem_ = this.getOptions()[0];
```

Comment:
It's possible to not have a selected menu item. I know this is harmless because you're just using it to scroll, but just so it's not confusing for anyone reading the code, set this here to null, and check for this.selectedMenuItem_ before scrolling.

Reply:
Be sure to annotate it to meet the style guide: 
```
  /**
   * A reference to the currently selected menu item.
   * @type {Blockly.MenuItem}
   * @private
   */
  this.selectedMenuItem_ = null;
```

Reply:
This makes sense! Thank you. I was confused on whether that commend should be within the  function or if it should be on the outside below all the Blockly.FieldDropdown.prototypes.